language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSPackageLoader.java

```diff
@@ -67,7 +67,8 @@ private LSPackageLoader(LanguageServerContext context) {
         List<Package> packages = new ArrayList<>();
         pkgMap.forEach((key, value) -> value.forEach(nameEntry -> {
             String[] components = nameEntry.split(":");
-            if (components.length != 2 || components[0].startsWith("lang.")) {
+            if (components.length != 2 || components[0].equals("lang.annotations")
+                    || components[0].equals("lang.__internal")) {
```

Comment:
shall we reverse the `equals` checks to avoid NPEs.

Reply:
I think the `components` array will not contain a null at any point, split API specifies that it will not.